apps/web/composables/useCustomer/useCustomer.ts

```diff
@@ -63,7 +63,13 @@ export const useCustomer: UseCustomerReturn = () => {
     useHandleError(error.value);
     state.value.data = data?.value?.data ?? state.value.data;
     checkUserState();
-    useWishlist().setWishlistItemIds(state.value.data?.basket?.itemWishListIds || []);
+    const wishlistIdsFromSession = state.value.data?.basket?.itemWishListIds || [];
+    const wishlistIdsJSON = JSON.parse(JSON.stringify(wishlistIdsFromSession));
+    const wishlistIds = [];
+    for (const wishlistObject of wishlistIdsJSON) {
+      wishlistIds.push(wishlistObject.variationId);
+    }
+    useWishlist().setWishlistItemIds(wishlistIds || []);
```

Comment:
Do we need to stringify, parse and then iterate over the response again ? Why doesn't it work without doing that ?